data-explorer/kusto/management/materialized-views/materialized-view-overview.md

```diff
@@ -177,6 +177,9 @@ The main contributors that can impact a materialized view health are:
 
 * **Materialized view definition**: The materialized view definition must be defined according to query best practices for best query performance. For more information, see [create command performance tips](materialized-view-create.md#performance-tips).
 
+* **Materialized view workload group limits**: By default materialized views are limited by the maximum amount of memory a query can allocate and by the percentage of nodes/threads on each node to fan out query execution to.
```

Comment:
I think this should be added as a sub-bullet under the *Cluster resources* bullet (first in this section). Consider something like: 
        `* By default, the [materialization process](link to how materialized views work) is limited by the amount of memory and CPU it can consume. These limits are defined, and can be changed, in the [materialized views workload group](link to the WG section).`